rvn-cli/rvn.go

```diff
@@ -452,3 +458,103 @@ var red = color.New(color.FgRed).SprintFunc()
 var redb = color.New(color.FgRed, color.Bold).SprintFunc()
 var yellow = color.New(color.FgYellow).SprintFunc()
 var bold = color.New(color.Bold).SprintFunc()
+
+func CheckRvnImages(topo rvn.Topo) {
+	var images []string
+	// NOTE: moving over to an extentionless naming scheme
+	// for each Node, get the image required
+	for i := range topo.Nodes {
```

Comment:
also need to iterate over switches

Reply:
At the moment the `Node` and `Zwitch` (switch) data structures are exactly the same and share a common composition `Host`, you can just smash the two lists together and iterate over hosts.

Reply:
I've got ghetto code doing this - will submit another commit to write it better.